packages/core/src/symbols/symbol-flow.ts

```diff
@@ -0,0 +1,94 @@
+import {
+  Context,
+  effect,
+  getContext,
+  onCleanup,
+} from "@alloy-js/core/jsx-runtime";
+import { isRef, Ref, shallowReactive } from "@vue/reactivity";
+import { MemberScopeContext, ScopeContext } from "../index.browser.js";
```

Comment:
is importing from .browser.js intentional?

Reply:
No sir, stupid vscode.